lib/cocoapods/installer/xcode/pods_project_generator/pod_target_integrator.rb

```diff
@@ -84,29 +74,33 @@ def add_copy_resources_script_phase(native_target, spec)
                 spec_paths_to_include = dependent_target.library_specs.map(&:name)
                 spec_paths_to_include << spec.name if dependent_target == target
                 dependent_target.resource_paths.values_at(*spec_paths_to_include).flatten.compact
-              end
+              end.uniq
+
               unless resource_paths.empty?
-                resource_paths_flattened = resource_paths.flatten.uniq
-                input_paths = [script_path, *resource_paths_flattened]
-                output_paths = UserProjectIntegrator::TargetIntegrator.resource_output_paths(resource_paths_flattened)
+                input_file_list_path = target.embed_frameworks_script_input_files_path_for_spec(spec)
```

Comment:
I missed this. This is wrong and needs to be `target.copy_resources_script_input_files_path_for_spec(spec)`

Fixing in a PR. This is affects the unreleased 1.7.0

Reply:
See https://github.com/CocoaPods/CocoaPods/pull/8222